Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py

```diff
@@ -0,0 +1,1152 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+import requests
+import json
+import os
+import time
+import sys
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+if not demisto.params().get('useProxy', False):
```

Comment:
use handle proxy,
notice that the parameter name is "useProxy" so pass it to the function:
`handle_proxy(proxy_param_name='useProxy')`